README.md

```diff
@@ -95,30 +95,12 @@ To uninstall:
 
 Distributed under the GPLv3 License. See `LICENSE` for more information.
 
-### Disclaimer
-
-- This fork has been created by the community to support the development of PlayCover, since the original project became non-free and non-open-source.
-- This fork is not affiliated with the original project, nor the original author.
-- This fork is not affiliated with the website <https://playcover.me>.
-- If you have any questions, please ask the community for help first.
-- The original project is under GNU General Public License v3.0, so there is no legal issue to fork it and redistribute.
-
-If anyone feels like this fork somehow violates the copyright (e.g., the logo), please open an issue.
-
-Many things are under construction, so please be patient. Any contribution is welcome.
-
-This fork will always stay open-source and maintained by the community.
-
-<p align="right"><a href="#top">⬆️ Back to top️</a></p>
-
 
 
 <!-- CONTACT -->
 ## Contact
 
-Lucas Lee - playcover@lucas.icu
```

Comment:
I think this was stated earlier. Discord is not a direct alternative to an email contact.

Reply:
Lucas is not a reliable contact. If we use an email for the contact info, it should be a different email. We could also use a Discord user id or something.

Reply:
I will discuss a possible email replacement or addition. A Discord username is not a replacement either. Inquiries of legal nature for instance would only be sent by email.